src/test/java/net/datafaker/AustraliaTest.java

```diff
@@ -9,8 +9,7 @@
 public class AustraliaTest extends AbstractFakerTest {
 
     @Test
-    public void locations() {
-        assertThat(faker.australia().locations(), not(is(emptyOrNullString())));
+    public void locations() { assertThat(faker.australia().locations(), not(is(emptyOrNullString())));
```

Comment:
Misformatted

Reply:
Done

Reply:
I made the corresponding format changes locally, please check my last commit to see if it's corrected. 

Apologies.